tests/prow_ci.sh

```diff
@@ -30,3 +30,147 @@ override_rosacli_build () {
   # go back to previous dir
   cd -
 }
+
+# setup aws credentials for the CI job run
+configure_aws () {
+  # configure aws region
+  if [[ -z "$REGION" ]]; then
+    REGION=${LEASED_RESOURCE}
+  fi
+  echo "[CI] Configured AWS region to ${REGION}"
+  # configure aws credetials
+  AWSCRED="${CLUSTER_PROFILE_DIR}/.awscred"
```

Comment:
The method with the hardcode variables depended too much on the Prow CI framework. When we call `configure_aws` in the build process, the better way should be `configure_aws $AWSCRED $REGION`

So are the below methods.

Reply:
Updated